SU-HSDO/suhumsci
# SHS-5623: Update styles for Date Stacked Vertical Card pattern

# READY FOR REVIEW

## Summary
Updates for Date Stacked Vertical Card and reuse code

## Need Review By (Date)
05/17

## Urgency
medium

## Steps to Test
1. Visit the following pages on Colorful and Traditional

-  `/mixed-pages/mixed-page-raised-cards`
-  `/collections-cards-continued`

2. Confirm the Date Stacked Vertical Card works and looks as expected on [Figma Option C](https://fourkitchens.clickup.com/t/36718269/SHS-5623#:~:text=UX/Design%20Tickets%20Work)
3. Since we are reusing code for `Vertical Card`, `Vertical Linked Card` and `Date Stacked Vertical Card` please, confirm all of them still look and work as they were before

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
